terminal.py

```diff
@@ -0,0 +1,57 @@
+import subprocess
+import os
+from pathlib import Path
+
+# Class to represent an instance of a terminal which is initiated
```

Comment:
In bot.py we have the method/class headers on the lines following the method/class, but here they are before. We need to pick one.

Reply:
Added to to-do list for 8/23